Tests/biosql.ini.sample

```diff
@@ -20,6 +20,6 @@ testdb=biosql_test
 [pg]
 # Covers DBDRIVER="psycopg2" etc
 dbhost=localhost
-dbuser=postgres
-dbpasswd=
+dbuser=biosql_user
+dbpasswd=biosql_pass
```

Comment:
This change means that the settings no long match those provided by TravisCI, so a note that the account is created in ``.travis.yml`` would be helpful.

Reply:
OK